client/src/components/Viewer/Viewer.js

```diff
@@ -55,6 +55,9 @@ function Viewer(props) {
         setIsConnected(true);
         socket.emit('addUser', {
           token: JSON.parse(localStorage.getItem('dfs-user'))?.['token'],
+          user_id: JSON.parse(localStorage.getItem('dfs-user'))?.['user'][
+            'user_email'
+          ],
```

Comment:
The `user_id` property is being set to the `user_email` from the parsed `localStorage` data. This could lead to confusion as `user_id` typically implies a unique identifier rather than an email. Consider renaming this property to `user_email` for clarity.
```diff
-           user_id: JSON.parse(localStorage.getItem('dfs-user'))?.['user'][
+           user_email: JSON.parse(localStorage.getItem('dfs-user'))?.['user'][
```
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*